src/components/LHNOptionsList/LHNOptionsList.js

```diff
@@ -136,13 +132,10 @@ function LHNOptionsList({
         ({item: reportID}) => {
             const itemFullReport = reports[`${ONYXKEYS.COLLECTION.REPORT}${reportID}`] || {};
             const itemReportActions = reportActions[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`];
-            const itemParentReportActions = reportActions[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${itemFullReport.parentReportID}`];
-            const itemPolicy = policy[`${ONYXKEYS.COLLECTION.POLICY}${itemFullReport.policyID}`];
-            const itemTransaction = `${ONYXKEYS.COLLECTION.TRANSACTION}${lodashGet(
-                itemParentReportActions,
-                [itemFullReport.parentReportActionID, 'originalMessage', 'IOUTransactionID'],
-                '',
-            )}`;
+            const itemParentReportActions = reportActions[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${itemFullReport.parentReportID}`] || {};
+            const itemPolicy = policy[`${ONYXKEYS.COLLECTION.POLICY}${itemFullReport.policyID}`] || {};
+            const itemTransactions = lodashGet(itemParentReportActions, [itemFullReport.parentReportActionID, 'originalMessage', 'IOUTransactionID'], '');
+            const transactionsList = itemTransactions ? transactions[`${ONYXKEYS.COLLECTION.TRANSACTION}${itemTransactions}`] : transactions;
```

Comment:
I am not sure if we want to pass the entire collection of transactions in this case if itemTransaction would not be found
```suggestion
            const transactionsList = itemTransaction ? transactions[`${ONYXKEYS.COLLECTION.TRANSACTION}${itemTransaction}`] : transactions;
```